Sources/NIOWebSocketServer/main.swift

```diff
@@ -145,7 +145,7 @@ private final class WebSocketTimeHandler: ChannelInboundHandler {
         buffer.write(string: "\(theTime)")
 
         let frame = WebSocketFrame(fin: true, opcode: .text, data: buffer)
-        ctx.writeAndFlush(self.wrapOutboundOut(frame)).map { (_: Void) in
+        ctx.writeAndFlush(self.wrapOutboundOut(frame)).map { _ in
```

Comment:
Let's not explicitly ignore the parameter: that's unsafe if we ever changed the return value of this future. Let's just do `{` instead.